src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -136,8 +136,6 @@ export default function withAnnotations<P extends object>(
         };
 
         handlePreviewDestroy = (): void => {
-            this.setState(defaultState);
```

Comment:
No longer needed? Does it get reset elsewhere?

Reply:
A full reset overrides the location changes made by the sidebar, particularly around changing versions to an active annotation id. Maybe I’ll reset other parts of the state but leave active annotation alone